### DISCUSSION TITLE
Is there literally ANY working version, anymore????

### DISCUSSION THREAD

**Discussion by Astral-Crescent** [victornpb/undiscord#556] [1. Questions & Answers]
I may have figured out my own solution. Ty for your time. Will come back with more info if I haven't fixed it.

**Comment by g-l-i-t-c-h-o-r-s-e** [comment]
> I may have figured out my own solution. Ty for your time. Will come back with more info if I haven't fixed it. 

Sure would be helpful to share your solution.

**Reply by Yaimsputnik5** [reply]
I've been using an older version of Undiscord that works just fine, you just have to hit downgrade.

https://github.com/victornpb/undiscord/blob/cad14199a51d7c725c980655a2a4d80bf5202f8f/deleteDiscordMessages.user.js
